jane/doc/extensions/_04-modes/syntax.md

```diff
@@ -24,19 +36,19 @@ Modes are in a dedicated namespace separated from variable names or type names,
 which means you can continue to use `local` or `unique` as variable or type
 names.
 
-Currently a modality expression is simply a space-delimited list of modalities.
+A mode expression actually contains a specification for each modal axis, whether
+you have written a choice for that axis or not. For axes that are omitted, the
+so-called *legacy* modes are used instead. The legacy modes are as follows:
 
+```ocaml
+global aliased many nonportable uncontended unyielding stateful read_write
 ```
```

Comment:
This is only true for modes on arrow types. But I think maybe better to move this to another document about semantics.

Reply:
Yeah when you write `(exp : _ @ local)` you are only specifying the locality; the rest are not constrained.

Reply:
Ah yes. I've revised. Thanks for catching this.